src/content/docs/logs/troubleshooting/no-log-data-appears-ui.mdx

```diff
@@ -43,6 +43,35 @@ If no data appears after you set up a log-reporting solution and wait about five
 These agents are designed to automatically report log data. This is referred to as [logs-in-context](/docs/logs/logs-context/logs-in-context). If you don't see that data, or want to do additional configuration, some options: 
 
 * APM: [manual processes for setting up log reporting](/docs/logs/logs-context/logs-in-context/#enable-logs)
```

Comment:
@homelessbirds I'm going to recommend you replace line 45 with this:


* Review our docs on setting up [APM logs-in-context](/docs/logs/logs-context/get-started-logs-context/)

Otherwise LGTM